src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -254,7 +254,7 @@ private UnboundLambda BindAnonymousFunction(AnonymousFunctionExpressionSyntax sy
                     var type = lambda.Data.ParameterTypeWithAnnotations(i);
                     if (type.HasType && type.IsStatic)
                     {
-                        Error(diagnostics, ErrorCode.ERR_ParameterIsStaticClass, syntax, type.Type);
+                        Error(diagnostics, ErrorFacts.GetStaticClassParameterCode(isInInterface: false), syntax, type.Type);
```

Comment:
nit: I'd probably would have kept using the error code directly, but that's not crucial...
Did you check we have existing tests for all the scenarios that are now using the two helper methods?

Reply:
I did check for the other scenarios. I wanted to use the new helper in all locations in order to make it obvious to future readers that we'd looked at everywhere that reported the warning and made an explicit decision, not that it was forgotten when implemented.

---
In reply to: [474303458](https://github.com/dotnet/roslyn/pull/47012#discussion_r474303458) [](ancestors = 474303458)